src/Login/Logout.js

```diff
@@ -7,6 +7,9 @@ const cookies = new Cookies();
 
 export default function Logout(){
     cookies.remove('AuthToken'); // determine if authorized, from context or however you're doing it
+    cookies.remove('user_mail');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9172775)
```suggestion
    cookies.remove("user_mail");
```